projects/plugins/jetpack/_inc/client/my-plan/my-plan-partner-coupon/index.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import MyPlanBanner from '../my-plan-banner';
+import { getPartnerCoupon } from 'state/initial-state';
+
+const MyPlanPartnerCoupon = ( { siteAdminUrl, partnerCoupon } ) => {
+	if ( 'object' !== typeof partnerCoupon ) {
+		return null;
+	}
+
+	const redeemButton = (
+		<Button primary href={ `${ siteAdminUrl }admin.php?page=jetpack&showCouponRedemption=1` }>
+			{ __( 'Redeem', 'jetpack' ) }
```

Comment:
The designs include the product name in the button so it's "Redeem Jetpack Backup"

Reply:
Oh yeah, I completely forgot to mention this in the description! The copy has been slightly changed because of a conversation that was going on for the JITM copy update (source: pdpAdu-1U-p2#comment-53).

I'm adding this for specific design approval in the PR description